packages/shared/src/utils/ssr.ts

```diff
@@ -1,3 +1,3 @@
 export const inClientSide = (): boolean => {
```

Comment:
🙃 `isClientSide` or `isCSR` ?

Feel free to keep it as is

Reply:
I prefer the longest one tbh